source/common/json/json_loader.cc

```diff
@@ -154,6 +155,33 @@ class ObjectImplBase : public Object {
 
   bool hasObject(const std::string& name) const override { return value_.HasMember(name.c_str()); }
 
+  void validateSchema(const std::string& schema) const override {
+    rapidjson::Document document;
+    if (document.Parse<0>(schema.c_str()).HasParseError()) {
+      throw std::invalid_argument(fmt::format("invalid schema \n Effor(offset {}) : {}\n",
+                                              document.GetErrorOffset(),
+                                              GetParseError_En(document.GetParseError())));
+    }
+
+    rapidjson::SchemaDocument schema_document(document);
```

Comment:
This can't be the case / doesn't make sense. You can have a valid JSON document which is not a valid schema. So there must be a way of catching an invalid schema error somewhere, at the very least when you do line 169. I would be shocked if there is no way to check that the schema is valid syntax.